types/js-cookie/js-cookie-tests.ts

```diff
@@ -19,8 +19,7 @@ Cookies.get();
 Cookies.remove('name');
 Cookies.remove('name', { path: '' });
 
-const Cookies2 = Cookies.noConflict();
-Cookies2; // $ExpectType CookiesStatic
+const Cookies2: Cookies.CookiesStatic = (Cookies.noConflict as () => Cookies.CookiesStatic)();
```

Comment:
It was much clearer before this change, why did you change it?

Reply:
Are you talking about that I removed the `$ExpectType`? Or that the noConflict method is now optional? 

I'll reintroduce the `$ExpectType`, thanks for explaining what it does.

Reply:
You can do the call like this, without explicit cast:
```typescript
const Cookies2 = Cookies!.noConflict();
```

Reply:
Thanks. I saw the bang operator back when it was introduced, loved it, then forgot about it. 

Btw. its like this:  `const Cookies2 = Cookies.noConflict!();`

Reply:
Oh, thanks!